package.json

```diff
@@ -21,7 +21,7 @@
     "eslint": "~3.7.0",
     "eslint-config-eslint": "^3.0.0",
     "mocha": "^2.4.5",
-    "typescript-eslint-parser": "^0.4.0"
+    "typescript-eslint-parser": "^1.0.0"
```

Comment:
@dannyfritz You are right, I was working on some other rule that required the upgrade.
I will send a different PR for this.